### ISSUE TITLE
Configuration Error

### ISSUE THREAD

**Issue by 73CH9ICH13F** [sithis993/Crypter#77]
Hello again. I fired up my VM and starting testing my Crypter. Without any Configuration and leaving all fields blank it works. Adding own configuration it throws up 2 errors. I translated ''Error 1'': **Crypter.exe - Invalid image 
IME140.dII is either not available for execution under 
Windows or contains an error. Install 
the program with the original installation media again, 
or contact your system administrator or 
software vendors to get support. 
** 
Error 2: LoadLibrary: Incorrect Parameter.

![1 Error](https://user-images.githubusercontent.com/30529878/81304289-28521080-907d-11ea-812d-9f1b3a0bce5d.PNG)
![2 Error](https://user-images.githubusercontent.com/30529878/81304283-26884d00-907d-11ea-98d8-212fc19cffee.PNG)

**Comment by sithis993** [comment]
Okay so I think this might be related to UPX... are you using UPX? if so can you try building without UPX and then running Crypter. I have a feeling this will fix the problem because there might be some kind of issue with UPX and some of the more recent versions of PyInstaller.

If this fixes the problem then I'll investigate and see if there's some kind of way to get UPX working again. At the moment I'm afraid this probably is the quickest way to contact me. I'm usually quite busy with my day job as well as a few other projects, so I'm not really available on Instant Messaging services :-(

**Comment by 73CH9ICH13F** [comment]
Alright. After removing UPX directory got this error message.
![Unbenannt](https://user-images.githubusercontent.com/30529878/81336124-03739280-90a9-11ea-9d78-f391ab45c95a.PNG)

**Comment by sithis993** [comment]
Hey dude, sorry it's been so long. Are you still having these problems?

**Comment by 73CH9ICH13F** [comment]
Sup man, yes I‘m still having these problems, have you pushed an update?

**Comment by sithis993** [comment]
I've pushed some updates yes, but not sure they will fix this problem.... Are you only hitting these problems when you use UPX? or can you not build and run Crypter at all?

**Comment by 73CH9ICH13F** [comment]
Nice. Haven’t tried in a while, but it definetly don‘t work with UPX.

**Comment by sithis993** [comment]
Hmm. Okay, let me give it a shot now. I think this is due to a PyInstaller compatability issue with UPX

**Comment by sithis993** [comment]
This is proving a bit more difficult than I expected.... I may need to take UPX support out for now. Are you having problems without UPX?

**Comment by 73CH9ICH13F** [comment]
I‘ll check if I still have problems with UPX.

**Comment by sithis993** [comment]
I think I may have just solved one of the problems... gimme a few mins

**Comment by sithis993** [comment]
Yup, I've figured out what's going on here. Working on fixing it at the minute and then will push up the fix. I won't create a new release because I created version 3.5 today. I'll stick it into that release and let you know when its there. Then you can give it a test :-).

It might not fix the pyiboot01_bootstrap error you were seeing, but it will enable you to use UPX at least. From there I can work on fixing the bootstrap error. I'm hoping that the UPX fix might fix both problems but we'll see.

**Comment by 73CH9ICH13F** [comment]
Alright, thanks for your work man.

**Comment by sithis993** [comment]
No problem. I've fixed this now and pushed the fix up to the develop branch. Can you give it a test and see if it works? :-). If it does, then I think I actually will create a new release (3.6). It's a bit easier than modifying 3.5.

Also, I've added something else which I think will really help with troubleshooting errors. In the "Binary Settings" section of the Builder there's now a "Debug Mode" checkbox. If you tick this and then build the exe and run it, a console window will open alongside the ransomware and any error or debug messages will be printed there.

This will mean you'll actually get a useful error message instead of the usual "Failed to execute main script" that PyInstaller displays

One caveat here though: You'll obviously only want to tick this box when there's actually a problem and you want a bit more information on what's going on. Otherwise the user will open the exe and then see the console window containing the debug, and probably get very suspicious. So, the general troubleshooting process would look like this:

1. Build an exe with Debug Mode disabled
2. An error occurs and you see "Failed to execute main script"
3. Rebuild the exe with Debug Mode enabled
4. Run the exe and check the console window to see what's going on

Make sense? :-) Hope that's not too convoluted hahaha

**Comment by 73CH9ICH13F** [comment]
I‘ll test it once I‘m home.

**Comment by sithis993** [comment]
Have you managed to give this a shot yet?

**Comment by ricko2991** [comment]
how to debug the exe file as you said?
I've tried it on the cmd.exe console but the error message doesn't appear.

**Comment by sithis993** [comment]
Hey Ricko. You might not see anything because the error you're having is a little bit different. It looks like it's some kind of system's problem, or an issue with the executable.

This new option is mainly for catching and logging any runtime exceptions of the Crypter code itself

**Comment by ricko2991** [comment]
hello, how can I debug my error for example like in cmd ?
